src/components/Sections/WidgetEmptyState.js

```diff
@@ -0,0 +1,12 @@
+import './WidgetEmptyState.less';
+import React from 'react';
+
+const EMPTY_STATE_TEXT = 'No results found';
```

Comment:
use emptyString first, 
and if no value use the const , 
and move the const to constants file